gamestonk_terminal/cryptocurrency/onchain/onchain_controller.py

```diff
@@ -77,7 +77,6 @@ class OnchainController:
         "th",
         "prices",
         "address",
-        "active",
```

Comment:
Was this on purpose?

Reply:
yea, when i moved active command from onchain to due dilligence menu i forgot to remove it from the commands list of onchain